neptune/experiments.py

```diff
@@ -518,14 +526,16 @@ def log_artifact(self, artifact, destination=None):
                           upload_tar_api_fun=self._client.extract_experiment_output,
                           experiment=self)
 
-    def download_artifact(self, filename, destination_dir):
+    def download_artifact(self, filename, destination_dir=None):
         """Download an artifact (file) from the experiment storage.
 
         Download ``filename`` from the experiment storage and save it in ``destination_dir``.
 
         Args:
             filename (:obj:`str`): Name of the file to be downloaded.
```

Comment:
`filename` can be a path, not just a filename. Let's change it to `path` for compatibility with download_sources/artifacts.